dashboard/views.py

```diff
@@ -33,8 +33,14 @@
 CANVAS_FILE_PREFIX = config("CANVAS_FILE_PREFIX", default="")
 CANVAS_FILE_POSTFIX = config("CANVAS_FILE_POSTFIX", default="")
 CANVAS_FILE_ID_NAME_SEPARATOR = "|"
+# This is fixed from UDW
+UDW_ID_PREFIX = config("UDW_ID_PREFIX", default="17700000000")
```

Comment:
I'd just use settings.UDW_ID_PREFIX for this value when needed. Better to just define everything in settings. We need to fix cron.py reusing this value too.

Reply:
I don't even see this used in this change?